modules/microshift-firewalld-install.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-firewall-install_{context}"]
+== Installing the `firewalld` service
+To install and run the `firewalld` service, run the following commands:
```

Comment:
I would suggest an overview of this procedure like...

Use the following procedure to install and run the `firewalld` service. 

Also, remove the backticks in the heading (`firewalld`). We don't put backticks in headings